### ISSUE TITLE
Color contrast ratio of "groupBox5" text is 4.087:1 which is less than 4.5:1: A11y_.NET CoreWinforms_Winforms_Containers _ColorContrast

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11445]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG1.4.3; #DesktopApp; #Win11; #FTP; #ColorContrast;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Containers_Controls_2:Testing the controls under the containers Tab" and hit ENTER key.
4. Winforms_Containers screen would appear.
5. Check the color contrast ratio of "groupBox5" text present under 'Empty_Groupbox_Setbackcolor_Green'.

### Actual Result:
Color contrast ratio of "groupBox5" text is 4.087:1 which is less than 4.5:1.

### Expected Result:
Color contrast ratio of "groupBox5" text should be greater than or equal to 4.5:1.

### User Impact:
Low vision users will not be able to see the text clearly and will not get information about it.

### Attachment
![11445_A11y_ NET CoreWinforms_Winforms_Containers _ColorContrast ](https://github.com/dotnet/winforms/assets/90901519/169f08de-c930-4ec4-aa85-cf6463eaa8a4)

**Comment by merriemcgaw** [comment]
This can be fixed by changing the colors in the sample app.

**Comment by Olina-Zhang** [comment]
@PoojaNamde The test app and SDK have been updated, please download the updated app and .NET 9.0 SDK to test. It should be as following:
![image](https://github.com/dotnet/winforms/assets/26474449/b7c33659-1f49-43fa-b574-e91495705d4a)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-05-24;

Verified the bug in latest environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment

![image](https://github.com/dotnet/winforms/assets/90901519/75791f3b-f982-4efe-9078-7d422a346827)
